dipy/align/imaffine.py

```diff
@@ -144,6 +144,8 @@ def set_affine(self, affine):
             self.affine_inv = None
             return
         try:
+            if np.isnan(np.sum(affine)):
```

Comment:
Using `sum` is faster than, e.g., `min` for finding nan's, according to this:
http://stackoverflow.com/questions/6736590/fast-check-for-nan-in-numpy